resources/views/content/scripts.twig

```diff
@@ -30,9 +30,8 @@
     });
 </script>
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/svg-injector/1.1.3/svg-injector.min.js"></script>
-{#    <script src="{{ plugin_path("Findologic") }}/js/dist/findologic_ceres.js"></script>#}
-<script src="https://localhost:8080/findologic_ceres.js"></script>
+    <script src="{{ plugin_path("Findologic") }}/js/dist/findologic_ceres.js"></script>
+{#<script src="https://localhost:8080/findologic_ceres.js"></script>#}
```

Comment:
Preferable handled by environment, please at least add TODO comment

Reply:
As discussed I've added a section to the `README.md`